indexers/staking-squid/src/utils/index.ts

```diff
@@ -39,3 +38,18 @@ export const operatorUID = (operatorId: number): string => `${operatorId}`;
 
 export const nominatorUID = (operatorId: number, account: string): string =>
   `${operatorId}-${account}`;
+
+export const bundleUID = (
+  domainId: number | string,
+  domainBlockHash: string
+): string => `${domainId}-${domainBlockHash}`;
```

Comment:
There can be multiple bundles per domain block, so I don't think this will give you a unique identifier.

Reply:
So what should I use as unique identifier?

Reply:
Maybe give the bundles an index within a block, similar to how [unique identifiers](https://wiki.polkadot.network/docs/build-protocol-info#:~:text=The%20correct%20way%20to%20uniquely,always%20uniquely%20identify%20a%20transaction.) should be created with extrinsics.

Reply:
This has been added